dsn.go

```diff
@@ -604,6 +609,8 @@ func parseDSNParams(cfg *Config, params string) (err error) {
 			} else {
 				cfg.ValidateDefaultParameters = ConfigBoolFalse
 			}
+		case "tracing":
```

Comment:
Where are we setting the default to error?

Reply:
In driver.go, https://github.com/snowflakedb/gosnowflake/blob/master/driver.go#L55